### DISCUSSION TITLE
Add numbers to the x/y lines in a game preview for debugging purposes

### DISCUSSION THREAD

**Discussion by alexandramartinez** [BattlesnakeOfficial/feedback#231] [Feature Requests]
I'm a very visual person so I refer to the game board a LOT to check my mistakes. I have to manually count the x/y coordinates to make sure the snake/food/etc is indeed in the position I have in my code.

It would be great if the numbers would at least be visible from the board. What would be even more awesome would be if I can hover a square and it shows me the x/y position automatically. That way I don't have to manually count it. But I'd be super grateful just by having the numbers to the sides of the board 😄 

Thank you in advance!

**Comment by bvanvugt** [comment]
This is a great idea! Maybe something can be toggled on/off within the playback settings.

**Comment by andrewdmaclean** [comment]
@alexandramartinez this is going to be on our list to discuss next week!

**Comment by coreyja** [comment]
Loved this idea and hacked on it today on stream! [I'll post a link to the VOD when I wrap up streaming for]

PR is here: https://github.com/BattlesnakeOfficial/board/pull/71

Screenshot: 
![image](https://user-images.githubusercontent.com/964771/226197763-9fae3349-35c4-4a4f-badb-efe5a56ed3b7.png)

**Comment by alexandramartinez** [comment]
In my screen I cannot change the checkbox for it :( - I cannot scroll-down

![image](https://user-images.githubusercontent.com/3922764/226990071-0f883373-d05f-4465-9462-cac91aded009.png)

**Reply by coreyja** [reply]
😢 

Will need to look into that for the embed!

I think I have a hack for ya tho!
If you open the `board.battlesnake.com` in its own tab (thats an embedded iframe in the screenshot) you can hopefully toggle the setting to true, and then it'll get saved in LocalStorage and work for in the UI things too!

Here is a random game URL extracted you can use (until it expires in a week or so lol)
[The board ui requires a game id to show the settings page]
https://board.battlesnake.com/?engine=https%3A//engine.battlesnake.com&game=2b8704e4-2bc5-4e4a-a064-909f16c808c0&hideLogo=true

I was able to get my play UI working after that, but let me know if you run into issues

![image](https://user-images.githubusercontent.com/964771/227008651-802c2eca-5bcc-4310-9132-63e1f73542d2.png)

**Reply by alexandramartinez** [reply]
the workaround worked! thank you

**Comment by bvanvugt** [comment]
This is now live and no longer experimental as of Aug 14, 2023. Marking resolved!
